test/features/install_dataindex.feature

```diff
@@ -17,3 +17,29 @@ Feature: Kogito Data Index
       }
     }
     """
+
+#####
+
+  @externalcomponent
+  @infinispan
```

Comment:
did not pay attention here but maybe keep also the `@persistence` tag that we use in other scenarios ?

Reply:
well, it depends on the meaning of `@persistence` tag:
Does it refer to testing of data being persisted?
Or does it refer to usage of Infinispan or other persistence provider?

Maybe we can add a list of available tags into readme together with description.

Reply:
good question :D

I was seeing it more like: `we use persistence in this scenario`
like we have the `@events` for the `we use events in this scenario`

For now it is only infinispan but maybe later it will be also other providers

Reply:
> Maybe we can add a list of available tags into readme together with description.

yeah we should add the definition of each tags as it seems we don't have the same definitions :)

Reply:
This scenario is a bit blurry for `@persistence` tag as we use Infinispan here, but we don't test persistence explicitly.

Reply:
ok, let's let it like that.
created https://issues.redhat.com/browse/KOGITO-2010 for a tags review :)